### PR TITLE
SystemTimers: Don't name non-constant variables as if they are constant

### PR DESCRIPTION
**Pull request by JosJuice**

### CODE REVIEW

File: Source/Core/Core/HW/SystemTimers.cpp [hunk 1]
```diff
@@ -79,23 +77,23 @@ static int et_IPC_HLE;
...
+// Feel free to experiment. Set them in Init below.
 
-	// This is completely arbitrary. If we find that we need lower latency, we can just
-	// increase this number.
-	IPC_HLE_PERIOD;
+// This is a fixed value, don't change it
+static int s_audio_dma_period;
+// This is completely arbitrary. If we find that we need lower latency,
+// we can just increase this number.
+static int s_ipc_hle_period;
```
Comment (BhaaLseN) [inline Source/Core/Core/HW/SystemTimers.cpp]: Hm, this one is only initialized (and used, hopefully) when ```SConfig::bWii``` is true. Not sure if thats a good or bad thing, but I can't think of a different way to doing this either atm.

**Comment by degasus** [comment]
LGTM

File: Source/Core/Core/HW/SystemTimers.cpp (hunk 1 above)
Comment (JosJuice) [inline Source/Core/Core/HW/SystemTimers.cpp]: Yep, it's only used when `bWii` is true. All code that uses `s_ipc_hle_period` explicitly checks `bWii` first, even `IPC_HLE_UpdateCallback`, which shouldn't get called if `bWii` is false.

**Comment by waddlesplash** [comment]
LGTM

**Pull request merged**